src/storage/redis_metadata.h

```diff
@@ -62,7 +62,7 @@ enum RedisCommand {
 };
 
 const std::vector<std::string> RedisTypeNames = {"none", "string", "hash",      "list",  "set",
-                                                 "zset", "bitmap", "sortedint", "stream"};
+                                                 "zset", "bitmap", "sortedint", "stream", "MBbloom--"};
```

Comment:
```suggestion
                                                 "zset", "bitmap", "sortedint", "stream", "bloomfilter"};
```

Reply:
> https://github.com/RedisBloom/RedisBloom/blob/master/src/rebloom.c#L1448

Redis seems to use `MBbloom--` as the type name of the bloom filter. Should we be consistent?

Reply:
Lets keep consistent with Redis

Reply:
got it.